docs/reference/hacl/signature/eddsa.md

```diff
@@ -19,16 +19,12 @@ Two APIs are exposed: A (simple) "One-Shot" API to sign/verify a single message
 
 **Example**
 
-<!--
```

Comment:
The position of the examples is a little incosistent. Sometimes it's before the API and sometimes after. Can we get this unified?

Reply:
Yeah. Not sure what is better. Do you have a preference?

Reply:
No strong opinion either but leaning towards examples first, too.